anaUltraScurve.py

```diff
@@ -284,56 +353,51 @@ def overlay_fit(VFAT, CHAN):
         pass
     pass
 
-canv = r.TCanvas('canv','canv',500*8,500*3)
-legend = r.TLegend(0.75,0.7,0.88,0.88)
-if not options.PanPin:
-    canv.Divide(8,3)
+def saveSummary(vSum, vSum2, name='Summary'):
```

Comment:
Let's move this definition to `anautilities.py` that way other analysis scripts can also take advantage of it.  Seems like this type of thing could be used in:

- `anaUltraLatency.py`
- `anaUltraScurve.py`
- `anaUltraThreshold.py`